### ISSUE TITLE
Disable status bar

### ISSUE THREAD

**Issue by Kristinita** [SublimeLinter/SublimeLinter3#563]
### 1. Request

It would be nice, if would be possible disable warnings/errors in status bar, if user select [**tooltip mode**](https://github.com/SublimeLinter/SublimeLinter3/pull/552).

### 2. Justification

If tooltips enabled, status bar messages not necessarily.

Errors/warnings for SublimeLinter have a long length and can occupy whole status bar. Example:

![Status bar](http://i.imgur.com/sjjN8tu.png)

In Sublime Text we don't have multi-line sidebar, that to see all and full status bar messages, see [**here**](https://github.com/SublimeTextIssues/Core/issues/1642) for details.

### 3. Desirable behavior

For example, user set in `SublimeLinter.sublime-settings` file:

```json
"disable_status_bar": true
```

SublimeLinter will no longer send messages to the status bar.

Thanks.

**Comment by FichteFoll** [comment]
See also my suggestion regarding multiple (different, non-exclusive) representations in #471.

**Comment by braver** [comment]
Simply being able to disable the status bar would be nice, independent of tooltips or other presentations of the errors.

**Comment by Kristinita** [comment]
**Status: :heavy_check_mark: Fixed for me** in SublimeLinter 4.3.2

For another users:

For customize SublimeLinter messages status bar you need to use `statusbar.messages_template` and `statusbar.show_active_linters` options. If you need disable all SublimeLinter messages from status bar, you need:

```json
{
    "statusbar.messages_template": "",
    "statusbar.show_active_linters": true
}
```

For more details see comments in [**`SublimeLinter.sublime-settings`**](https://github.com/SublimeLinter/SublimeLinter/blob/master/SublimeLinter.sublime-settings) file.

Thanks.
